pkgs/applications/networking/cluster/werf/default.nix

```
@@ -4,30 +4,32 @@
 , fetchFromGitHub
 , installShellFiles
 , btrfs-progs
-, glibc
 , testers
 , werf
 }:
 
 buildGoModule rec {
   pname = "werf";
-  version = "1.2.168";
+  version = "1.2.171";
 
   src = fetchFromGitHub {
     owner = "werf";
     repo = "werf";
     rev = "v${version}";
-    hash = "sha256-/Shmnnpme1ffN7GMTryb4ddPlcAsruyWhFdjr1PJ3HM=";
+    sha256 = "sha256-SsBskEGKAHnTHyUzutVbownpYcd9ISLKZdbe67YCQrk=";
   };
 
-  vendorHash = "sha256-E5yDk48O7zze8QTeLQ999QmB8XLkpKNZ8JQ2wVRMGCU=";
+  vendorSha256 = "sha256-T+/7d5eS2iyYixgwKyT7T6SxSo+bQX3atiw4U0ypcvk=";
 
   proxyVendor = true;
 
   subPackages = [ "cmd/werf" ];
 
   nativeBuildInputs = [ installShellFiles ];
-  buildInputs = lib.optionals stdenv.isLinux [ btrfs-progs glibc.static ];
+
+  buildInputs = [ ]
+    ++ lib.optionals stdenv.isLinux [ btrfs-progs ]
+    ++ lib.optionals (stdenv.hostPlatform.libc == "glibc") [ stdenv.cc.libc.static ];
```

```suggestion
    ++ lib.optionals stdenv.hostPlatform.isGnu [ stdenv.cc.libc.static ];
```
Would that be equivalent?